tests/test_gcm.py

```diff
@@ -183,13 +190,14 @@ def test_fcm_options(self):
         Tests that the config option `fcm_options` allows setting a base layer
         of options to pass to FCM, for example ones that would be needed for iOS.
         """
-        gcm = self.sygnal.pushkins["com.example.gcm.ios"]
+        gcm = self.get_test_pushkin("com.example.gcm.ios")
         gcm.preload_with_response(
             200, {"results": [{"registration_id": "spqr_new", "message_id": "msg42"}]}
         )
 
         resp = self._request(self._make_dummy_notification([DEVICE_EXAMPLE_IOS]))
 
         self.assertEqual(resp, {"rejected": []})
+        assert gcm.last_request_body is not None
```

Comment:
I was going to say:

> I wonder if we can use [`self.assertIsNotNone`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNotNone) here, so that the assertion is more formally part of the test?

But I don't think that mypy will recognise this as narrowing `gcm.last_request_body` to something that's not-None. So probably best to ignore this comment and leave it as it is.